### PR TITLE
feat(event): add NodeCount, ReplicaCount, VolumeCapacity event fields

### PR DESCRIPTION
**Pull request by niladrih**
There were two sets of values sent for two different sets of engines:
1. cStor, Jiva --> Replica Count, Capacity
2. Zfs/LVM/Hostpath LocalPV --> "Running", Node count

It was either or for these values. The Event Category, Action, Label and Value stucture was ported directly from Google Analytics UA's event structure.

This PR removes Event Action, Label and Value and replaces them with 'NodeCount', 'ReplicaCount' and 'VolumeCapacity'. These 3 fields sit directly on the event body (similar to node arch and node OS).

### CODE REVIEW

**Review by Abhinandan-Purkait — APPROVED** [review]

**Comment by niladrih** [comment]
@Abhinandan-Purkait -- Thank you for your review! I've asked you for a second review because I've changed the implementation from what it was when you had reviewed it.

**Comment by Abhinandan-Purkait** [comment]
Looks, good. Is it a good idea to collect the sc params as a field as well? Maybe call that field `ExtraInfo` can be a generic field to add extra stuffs whenever we want. One immediate example could be on volume_provision events we set ExtraInfo to sc params. That would help us track what kind of configs people use.

**Review by Abhinandan-Purkait — APPROVED** [review]

**Comment by niladrih** [comment]
> Looks, good. Is it a good idea to collect the sc params as a field as well? Maybe call that field `ExtraInfo` can be a generic field to add extra stuffs whenever we want. One immediate example could be on volume_provision events we set ExtraInfo to sc params. That would help us track what kind of configs people use.

This needs more discussion. This might be a good item for the next maintainers' meeting. I'm going to hold off on this change for now.

**Review by Abhinandan-Purkait — APPROVED** [review]

**Pull request merged**
